### DISCUSSION TITLE
Sound

### DISCUSSION THREAD

**Discussion by thniel** [Kemmey/Kemmey-TeslaWatch-Public#556] [General]
Using the Watch fro Tesla app, can the sound from the Apple Watch be heard on the Tesla speakers?
Since its possible to access Chat-GPT from Apple Watch using voice-to-text input, it would be a great feature if it was possible to hear the answer from Chat-GPT (voice by Siri) on the speakers of the Tesla, since the sound on the watch is hard to hear through the noice of the car and traffic.

**Comment by Kemmey** [comment]
Yeah - I would love to be able to fix that - but alas, it is not possible as far as I know. The watch does not want to pair to the vehicles Bluetooth audio interface, cause the vehicle does not report as a pair of headphones. If it did, it would work. It's a really weird decision on apples part, that the watch does not want to Bluetooth pair with a normal audio component.
